Samples/SqueezeNetObjectDetection/Desktop/cpp/main.cpp

```diff
@@ -153,7 +158,32 @@ void LoadLabels()
     }
 }
 
-VideoFrame LoadImageFile(hstring filePath)
+ColorManagementMode GetColorManagementMode(const LearningModel& model)
+{
+    // Get model color space gamma
+    hstring gammaSpace = L"";
+    try
+    {
+        gammaSpace = model.Metadata().Lookup(L"Image.ColorSpaceGamma");

```

Comment:
`Keys and values are case insenstive` according to https://github.com/onnx/onnx/blob/master/docs/MetadataProps.md , so  e.g.`image.colorspacegamma` is valid too. The same goes for e.g. `sRGB` instead of `SRGB`.

Reply:
Sounds good! Good catch :)

Reply:
Is this resolved? Is the `Lookup` function already case-insensitive?

Reply:
model.Metadata() is a IMapView interface so it'll be up to WinML to handle the case-insensitive key lookup :)

Reply:
"Image.ColorSpaceGamma" should return the same value as "image.ColorSpaceGamma"

Reply:
Okay then :)